Automattic/wp-calypso
# Show `Add Review` button on theme details pages

Closes https://github.com/Automattic/dotcom-forge/issues/5145

## Proposed Changes

* Remove `marketplace-add-review` feature flag
* Remove feature flag guard from `Add Review` button on theme details pages

## Testing Instructions



On wpcalypso or dev environments:
- go to a theme details page with the feature flag disabled, eg. `/theme/solarone?flags=-marketplace-add-review`
- check that the ratings and `add a review` buttons are shown in the theme header
  ![CleanShot 2024-01-17 at 16 59 52@2x](https://github.com/Automattic/wp-calypso/assets/11555574/6d370ec1-ccc1-43c6-bf31-ce9f64fbcd91)


## Pre-merge Checklist
